### PR TITLE
Update test plan with unified resource tasks

### PR DESCRIPTION
**Pull request by avatus**
I made some small updates and added some unified resource specific tasks. A few of the tasks (the icons) require some pending PRs but would be good to include.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/webtestplan.md [hunk 1]
```diff
@@ -25,27 +25,33 @@ For main, test with a role that has access to all resources.
...
-
+- [ ] Verify that clicking on `Add Resource` button correctly sends to the resource discovery page
 #### Applications
-- [ ] Verify that clicking on `Add Application` button renders dialogue
-  - [ ] Verify input validation (prevent empty value and invalid url)
-  - [ ] Verify after input and clicking on `Generate Script`, bash command is rendered
-  - [ ] Verify clicking on `Regenerate` button regenerates token value in bash command
-
+- [ ] Verify that the app icons are correctly displayed
+- [ ] Verify that filtering types by Application includes databases in the page
```
Comment (zmb3) [inline .github/ISSUE_TEMPLATE/webtestplan.md:40]: Copy paste error?

File: .github/ISSUE_TEMPLATE/webtestplan.md [hunk 2]
```diff
@@ -25,27 +25,33 @@ For main, test with a role that has access to all resources.
...
+- [ ] Verify that the `Launch` button for applications correctly send to the app
+- [ ] Verify that the `Launch` button for AWS apps correctly renders an IAM role selectio window
```
Comment (zmb3) [inline .github/ISSUE_TEMPLATE/webtestplan.md:42]:
```suggestion
- [ ] Verify that the `Launch` button for AWS apps correctly renders an IAM role selection window 
```

File: .github/ISSUE_TEMPLATE/webtestplan.md [hunk 3]
```diff
@@ -25,27 +25,33 @@ For main, test with a role that has access to all resources.
...
+- [ ] Verify that the app icons are correctly displayed
+- [ ] Verify that filtering types by Application includes databases in the page
+- [ ] Verify that the `Launch` button for applications correctly send to the app
+- [ ] Verify that the `Launch` button for AWS apps correctly renders an IAM role selectio window 
 #### Databases
-- [ ] Verify that clicking on `Add Database` button renders dialogue for manual instructions:
-  - [ ] Verify selecting different options on `Step 4` changes `Step 5` commands
+- [ ] Verify that the database subtype icons are correctly displayed
+- [ ] Verify that filtering types by Databases includes databases in the page
+- [ ] Verify that clicking `Connect` renders the dialog with correct information
```
Comment (zmb3) [inline .github/ISSUE_TEMPLATE/webtestplan.md:46]: Is this something we can write a regular automated test for?

Comment (avatus) [reply .github/ISSUE_TEMPLATE/webtestplan.md:46]: True. We technically already have a test for it in `ConnectDialog.test.tsx` but it just felt right to test the buttons for the different types

**Review by mdwn — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/webtestplan.md [hunk 4]
```diff
@@ -25,27 +25,33 @@ For main, test with a role that has access to all resources.
 - [ ] Verify that Collapse/Expand works and collapsed has icon `>`, and expand has icon `v`
 - [ ] Verify that it automatically expands and highlights the item on page refresh
 
+### Unified Resources
+- [ ] Verify that scrolling to the bottom of the page renders more resources
+- [ ] Verify that all resource types are visible if no filters are present
 #### Servers aka Nodes
 - [ ] Verify that "Servers" table shows all joined nodes
```
Comment (kimlisa) [inline .github/ISSUE_TEMPLATE/webtestplan.md:32]:
```suggestion
- [ ] Verify that "Servers" type shows all joined nodes
```

File: .github/ISSUE_TEMPLATE/webtestplan.md [hunk 5]
```diff
@@ -25,27 +25,33 @@ For main, test with a role that has access to all resources.
...
 - [ ] Verify that "Connect" button shows a list of available logins
 - [ ] Verify that "Hostname", "Address" and "Labels" columns show the current values
 - [ ] Verify that "Search" by hostname, address, labels works
```
Comment (kimlisa) [inline .github/ISSUE_TEMPLATE/webtestplan.md:35]: are we keeping these? otherwise this section needs udpating?

**Review by kimlisa — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/webtestplan.md (hunk 5 above)
Comment (avatus) [reply .github/ISSUE_TEMPLATE/webtestplan.md:35]: i made a more generic task in the unified section and removed these. thanks!

**Pull request merged**